site/smart-accounts/accounts/light-account.md

```diff
@@ -20,33 +20,55 @@ head:
 
 # Light Account Smart Contract
 
-Light Account is a secure, gas-optimized, [ERC-4337](https://eips.ethereum.org/EIPS/eip-4337) smart account implementation.
+### Secure, optimized, and extendable
 
-We started with the Ethereum Foundation’s canonical [SimpleAccount](https://github.com/eth-infinitism/account-abstraction/blob/develop/contracts/samples/SimpleAccount.sol) and added key improvements for production app developers:
+## Overview
 
-- significantly reduced gas costs
-- [ERC-1271](https://eips.ethereum.org/EIPS/eip-1271) signature support to ensure users can sign messages, such as on OpenSea
-- ownership transfer so that users won’t get locked into a single `Signer`
+Light Account is a secure, gas-optimized, and versatile Ethereum smart account solution. It introduces efficiency, trust, and security to your web3 experience. In this document, we'll explore why Light Account is a valuable choice for developers.
+
+## Why Light Account
+
+Light Account is designed to meet your web3 needs, offering the following benefits:
```

Comment:
"designed to meet your web3 needs", it sounds to high level. Can we be more specific? 

"is a fully functioning web3 account, via smart contracts. It offers the following benefits"